crates/wmemcheck/src/lib.rs

```diff
@@ -101,12 +119,12 @@ impl Wmemcheck {
                         len: len,
                     });
                 }
-                MemState::ValidToWrite => {
+                /* MemState::ValidToWrite => {
```

Comment:
Why is this commented out? (If the checks were causing issues, could we put the exclusion under an appropriately named and documented option?)